pkg/exporter/process.go

```diff
@@ -235,6 +235,7 @@ func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 	bytesSlice := append(msg.GetMsgBuffer().Bytes(), set.GetBuffer().Bytes()...)
 	// Send the message on the exporter connection.
 	bytesSent, err := ep.connToCollector.Write(bytesSlice)
+
```

Comment:
is this required? L237 and L238 belongs to same code block.

Reply:
it is not needed. removed.